### ISSUE TITLE
DDR Menu doesn't handle duplicate page names correctly

### ISSUE THREAD

**Issue by daguiler** [dnnsoftware/Dnn.Platform#4483]
## Description of bug
Provide a clear and concise description of the bug.

## Steps to reproduce
1) Create following page hierarchy:
- Home
- ⤷ DDRMenu Test
- DDRMenu Test
- Some Other Page
- And Yet Another Page

the key here is to have 2 pages with identical names (at different branches of the hierarchy of course)

2) Change the DDR Menu tag in Portals\_default\Skins\Xcillion\Home.ascx to look like this:
```
<dnn:MENU ID="MENU" MenuStyle="Menus/MainMenu" runat="server" NodeSelector="*" IncludeNodes="Home, DDRMenu Test"></dnn:MENU>
```
(notice we are including only "Home" and "DDRMenu Test")

3) Refresh the home page
## Current behavior
The menu shows the "Home" link, and only 1 of the "DDRMenu Test" pages:

![image](https://user-images.githubusercontent.com/30123437/107636813-f5602d00-6c4b-11eb-897c-d59947962dcf.png)

## Expected behavior
The menu should display both instances of the duplicate page name:

![image](https://user-images.githubusercontent.com/30123437/107636835-fe50fe80-6c4b-11eb-9290-593d0a5c3f79.png)

## Affected version
* [X] 10.00.00 alpha build
* [X] 09.09.00 release candidate
* [X] 09.08.01 latest supported release

## Affected browser
* [X] Chrome
* [X] Firefox
* [X] Safari
* [X] Internet Explorer 11
* [X] Microsoft Edge (Classic)
* [X] Microsoft Edge Chromium

**Comment by Timo-Breumelhof** [comment]
@david-poindexter is there a way to avoid issues with a PR being picked up by stale bot?
This is in a way "finished" but just pushed to 10.. would be a pity if it would get closed..

**Comment by bdukes** [comment]
Yes, we can add the _On Hold_ label which will keep stalebot from processing it.

**Comment by Timo-Breumelhof** [comment]
Thank you @bdukes :-)

**Comment by bdukes** [comment]
FYI, I checked and I was wrong, _On Hold_ doesn't exempt an issue from stalebot, here's that list: https://github.com/dnnsoftware/Dnn.Platform/blob/91d3ead2053fe3e5a12bdd23474e5d91928d9ce1/.github/stale.yml#L5-L13

**Comment by Timo-Breumelhof** [comment]
ok thank you

**Comment by valadas** [comment]
Well I don't mind too much but semantically this is ready for development, it just goes stale because nobody submitted a PR to fix it, which is kind of the purpose of implementing stalebot in the first place. I'll make it a bug and ready for dev. "bug" should keep it here.

**Comment by bdukes** [comment]
This does have a PR (#4484), it's just waiting to be merged since it's a breaking change, AFAIK.

**Comment by valadas** [comment]
You are correct, I also added the `noteworth` tag to the PR so we don't forget to mention this behavior change.

**Comment by david-poindexter** [comment]
Closing because #4484 has been merged into the `release/10.0.0` branch.
